kcidb/test_db.py

```diff
@@ -18,6 +19,17 @@ def test_schemas_main():
     assert_executes("", *argv,
                     stdout_re=r"4\.0: 4\.0\n4\.1: 4\.1\n")
 
+def test_reset(clean_database):
+    """Check Client.reset  works"""
```

Comment:
Could you add the parens after `Client.reset` to signify it's a function/method (so it becomes `Client.reset()`), and also remove the extra space before "works"? So that the complete docstring becomes `"""Check Client.reset() works"""`?